src/components/Sponsors/style.css

```diff
@@ -0,0 +1,43 @@
+#sponsors-wrapper {
+    margin: 20px 120px;
+}
+
+#sponsors, #partners {
+    font-size: 64px;
+}
+
+#sponsor-us {
+    font-family: "Mulish";
+    font-size: 25px;
+}
+
+#email {
+    color: #F85E5E;
+    font-weight: bold;
+    font-family: "Mulish";
+}
+
+#partner-images, #sponsor-images {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+#sponsor-images img {
+    height: 125px;
+    width: auto;
+    margin: 20px;
+}
+
+#partner-images {
+    margin: 40px 50px;
+}
+
+#partner-images img {
+    height: 100px;
+    margin: 10px;
+    width: auto;
```

Comment:
<img width="639" alt="Screen Shot 2022-02-07 at 7 37 42 PM" src="https://user-images.githubusercontent.com/33165426/152913492-c7585717-ee14-45d9-9e72-336d02e85527.png">

I think it looks a little better doing if we change to fixed width and auto height so the number of logos per row is consistent (for sponsor and partner images)

```
{
    height: auto;
    margin: 1rem;
    width: 15rem;
}
```
(rem units are an alternative to pixels that scale a little bit better when users zoom in/out. 1rem = 16px)